requirements/requirements-py36-win.yml

```diff
@@ -0,0 +1,55 @@
+channels:
+  - conda-forge
+dependencies:
+  - python=3.6
+  - pandas
+  - numpy>=1.18.5
+  - pyarrow>=4.0.1
+  - dask[complete]>=2.22.0,<2021.3.0
+  - distributed>=2.22.0,<2021.3.0
+  - pickle5 # for dask to correctly serialize nested functions
+  - fsspec
+  - xarray
+  - Jinja2
+  - scipy
+  - pip
+  - s3fs>=2021.8,<=2022.1.0
+  - feather-format
+  - lxml
+  - openpyxl
+  - xlrd
+  - matplotlib<=3.2.2
+  - sqlalchemy>=1.4.0
+  - pandas-gbq
+  - pytables
+  - msgpack-python
+  - psutil
+  - pytest>=6.0.1
+  - pytest-benchmark
+  - pytest-cov>=2.10.1
+  - pytest-xdist>=2.1.0
+  - packaging
+  - coverage<5.0
+  - pygithub==1.53
+  - rpyc==4.1.5
+  - cloudpickle
+  - boto3
+  - scikit-learn
+  - pymssql
+  - psycopg2
+  - pywinpty>=1.1.0 # modin-spreadsheet needs that through jupyter, but latest versions won't install on Py36 via pip
```

Comment:
The only one difference from other .yml is `pywinpty` dependency. Could we process installing of this package depending on OS? In this case we will have only one .yml file for py36.

Also, I think that name containing `environment` word will be more habitual for conda env files (smth like `environment-py36.yml`).

Reply:
I wasn't able to find an ability to introduce logic in those `.yml` files, hence the (unfortunate) copy... maybe I should try pinning `jupyter` via `conda` instead though?..

Reply:
Ok, pinning `jupyter` via `conda` looks good.

What do you think about renaming which I touched in the comment above?

Reply:
Hmm, okay, let's rename.